src/main/java/me/zeroeightsix/kami/module/modules/chat/LoginMessage.kt

```diff
@@ -40,16 +40,14 @@ class LoginMessage : Module() {
             return
         }
         MessageSendHelper.sendChatMessage("$chatName Found '&7loginmsg.txt&f'!")
-
     }
 
-    @EventHandler
-    private val localPlayerUpdateEvent = Listener(EventHook { event: LocalPlayerUpdateEvent? ->
```

Comment:
Can you delete LocalPlayerUpdateEvent and where it posts the event in ForgeEventProcessor please?